packages/apollo-language-server/src/engine/operations/listServices.ts

```diff
@@ -0,0 +1,21 @@
+import gql from "graphql-tag";
+
+export const LIST_SERVICES = gql`
+  query ListServices($id: ID!, $graphVariant: String!) {
+    service(id: $id) {
+      implementingServices(graphVariant: $graphVariant) {
+        __typename
```

Comment:
I think we can leave `__typename` out everywhere, should be taken care of for us by our tooling.
(auto-added in our requests, as well as our generated types)